refinery/tool_manager/tests.py

```diff
@@ -3260,3 +3298,91 @@ def test_valid_tool_launch_config_PAIR_LIST(self):
             )
         }
         validate_tool_launch_configuration(tool_launch_configuration)
+
+
+class ToolManagerUtilitiesTests(ToolManagerTestBase):
+    def test_file_type_validation_error(self):
+        bad_filetype = "COFFEE"
+        error_message = "FileType `{}` does not exist".format(bad_filetype)
+
+        file_type_validation_error = FileTypeValidationError(
+            bad_filetype,
+            error_message
+        )
+        self.assertIn(bad_filetype, file_type_validation_error.message)
+        self.assertIn(error_message, file_type_validation_error.message)
+        self.assertIn(
+            str([f.name for f in FileType.objects.all()]),
+            file_type_validation_error.message
+        )
+
+    def test_get_visualization_annotations_list(self):
+        settings.VISUALIZATION_ANNOTATION_BASE_PATH = os.path.abspath(
+            os.getcwd()
+        )
+        tool_definition_name = "dummy.json"
```

Comment:
It looks like this test writes to the actual directory where tool defs are stored? ie, if the user actually had a `dummy.json`, it would be clobbered? Would it be possible to mock the internal setting for this dir, and have it point to a scratch space for the duration of the test?

Not sure if that makes sense.

Reply:
I change that behavior (`"writes to the actual directory where tool defs are stored"`) when I overwrite the `settings.VISUALIZATION_ANNOTATION_BASE_PATH`  at the beginning of this test